kratos/python_scripts/python_solver.py

```diff
@@ -179,21 +179,22 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
                 tmp = KratosMultiphysics.Parameters("{}")
                 KratosMultiphysics.ReorderAndOptimizeModelPartProcess(model_part, tmp).Execute()
 
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading model part from mdpa file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Finished reading model part from mdpa file.")
 
         elif (input_type == "rest"):
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Loading model part from restart file.")
             RestartUtility(model_part, self._GetRestartSettings(model_part_import_settings)).LoadRestart()
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Finished loading model part from restart file.")
 
         elif(input_type == "use_input_model_part"):
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Using already imported model part - no reading necessary.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Using already imported model part - no reading necessary.")
```

Comment:
why do you change this?
If it is changed then you no longer know where the print comes from

Reply:
sorry, you're right